plugins/harness-iacm/src/components/WorkspaceList/index.tsx

```diff
@@ -0,0 +1,203 @@
+/* eslint-disable @backstage/no-undeclared-imports */
+import React, { useState } from 'react';
+import {
+  CircularProgress,
+  FormControl,
+  FormHelperText,
+  InputLabel,
+  ListSubheader,
+  makeStyles,
+  MenuItem,
+  Select,
+} from '@material-ui/core';
+// eslint-disable-next-line no-restricted-imports
+import { Grid } from '@mui/material';
+import { discoveryApiRef, useApi } from '@backstage/core-plugin-api';
+import { AsyncStatus } from '../../types';
+import useGetResources from '../../hooks/useGetResources';
+import useProjectUrlSlugEntity from '../../hooks/useProjectUrlEntity';
+import { useEntity } from '@backstage/plugin-catalog-react';
+import { useResourceSlugFromEntity } from './useResourceSlugFromEntity';
+import ResourceTable from '../ResourceTable';
+
+const useStyles = makeStyles(theme => ({
+  container: {
+    width: '100%',
+  },
+  label: {
+    marginBottom: '2px',
+    fontSize: '1rem !important',
+    fontWeight: 400,
+  },
+  empty: {
+    padding: theme.spacing(2),
+    display: 'flex',
+    justifyContent: 'center',
+  },
+  select: {
+    border: '1px solid gray ',
+    paddingLeft: '5px',
+    height: '56px',
+    fontWeight: 400,
+    fontSize: '1rem',
+    color: 'rgba(0, 0, 0, 0.87)',
+    borderRadius: '4px',
+  },
+  menuItem: {
+    paddingLeft: '10px',
+    display: 'flex',
+    alignItems: 'center',
+    gap: '5px',
+  },
+}));
+
+export interface Workspace {
+  identifier: string;
+  name: string;
+}
+
+function WorkspaceList() {
+  const [refresh, setRefresh] = useState(false);
+
+  const classes = useStyles();
+  const discoveryApi = useApi(discoveryApiRef);
+  const backendBaseUrl = discoveryApi.getBaseUrl('proxy');
+
+  const { isWorkspaceAnnotationPresent, harnessWorkspaceUrlObject } =
+    useProjectUrlSlugEntity();
+
+  const allEnvsAnnotations = [
+    ['prod', 'harness.io/project-url'],
+    ['stage', 'harness.io/project-url-stage'],
```

Comment:
```suggestion
```
we no longer support stress or stage its all on devspaces now

Reply:
removed, thanks